app/javascript/dashboard/components/widgets/conversation/ReplyBox.vue

```diff
@@ -372,6 +374,19 @@ export default {
       }
     },
     message(updatedMessage) {
+      if (this.hasFiles === true) {
```

Comment:
This change will fix showing the image in the advance editor. @nithindavid @pranavrajs @fayazara Is there any other good way to solve this issue?